### DISCUSSION TITLE
Module parse failed: Unexpected token

### DISCUSSION THREAD

**Discussion by HenBaliti** [twilio-professional-services/flex-project-template#120] [Q&A] [answered]
Hello ,

I have been  working with twilio platform for a long time, I managed to build a lot of different plugins.

we wanted to upgrade all our plugins to "flex version 2" , I wanted to install the plugin of the new "flex version 2".
For this purpose, I did everything according to the above guide, I installed all the appropriate things (in terms of all versions, everything is compatible) but for some reason when I tried to run the plugin, I received a certain error:

```
Uncaught Error: Module parse failed: Unexpected token (1:320)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file.
```

It is important to note that I am not the only one who failed to pick up the plugin... a colleague from my work also tried and failed.

I have also attached some images from the console of the flex system , you can see the flex is running well (just the ui) but everything is "freeze" and not clickable...
![Screen Shot 2022-12-20 at 15 03 53](https://user-images.githubusercontent.com/58342958/208674727-acb516a4-b4b1-4723-883d-b67fe704a646.png)
![Screen Shot 2022-12-20 at 15 04 01](https://user-images.githubusercontent.com/58342958/208674734-6322333e-9597-4cc6-b567-5d67e9de575b.png)
![Screen Shot 2022-12-20 at 15 04 19](https://user-images.githubusercontent.com/58342958/208674738-24f0f627-d6c6-49a6-a0da-87094a88d0d4.png)

**Comment by dremin** [comment]
Are you updating from an earlier version of this template plugin? This looks like an issue with the Paste dependency, which was recently upgraded. Removing `node_modules` and `package-lock.json` then running `npm i` again in the plugin directory should result in the correct dependencies (Flex UI also depends on Paste, so upgrading the dependency in-place without these steps can sometimes run into conflicts).

**Reply by dremin** [reply, marked as answer]
Actually, I think there is an issue with a dependency, as I was able to reproduce this today. It looks like `@twilio-paste/icons@9.4.1` is problematic, if you install version 9.4.0 of that dependency it should clear things up. Would love to hear back if this works for you!
